nebari-dev/nebari
# Dashboard fix usage report

## Reference Issues or PRs


Fixes https://github.com/nebari-dev/nebari/issues/2591

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ X ] Bug fix (non-breaking change which fixes an issue)

## Testing

- [ X ] Did you test the pull request locally?

## How to test this PR?


Deploy nebari with this commit - check Grafana Dashboard "Usage Report" and you should see data (note that at least one JuptyerLab user environment must be running to see data)

## Any other comments?
